.github/workflows/pkgdown.yaml

```diff
@@ -6,6 +6,7 @@ on:
       - develop
       - feature/polish-vignettes
       - feature/update-pkgdown-settings
+      - feature/intro-vig
```

Comment:
Before merging, this line should be removed (I'll do it myself).
It is needed to build **pkgdown** website for the branch `feature/intro-vig`.